### PR TITLE
convert topology-helm-workload sidepanel to use dynamic extensions

### PR DESCRIPTION
**Pull request by sahil143**
fixes: https://issues.redhat.com/browse/ODC-5846, https://issues.redhat.com/browse/ODC-6109

Screenshot:
Screenshot 2021-09-06 at 1 52 49 PM
Screenshot 2021-09-06 at 1 52 58 PM

### CODE REVIEW

File: frontend/packages/topology/console-extensions.json [hunk 1]
```diff
@@ -122,7 +122,7 @@
   {
     "type": "console.topology/details/resource-link",
     "properties": {
-      "priority": 50,
+      "priority": 0,
```
Comment (invincibleJai) [inline frontend/packages/topology/console-extensions.json:125]: won't be good to have priority as greater than 0 , even if it's lowest

**Comment by invincibleJai** [comment]
/approve

verified the changes

**Review by rohitkrai03 — APPROVED** [review]
/lgtm
/approve

**Comment by rohitkrai03** [comment]
/label qe-approved

Verified and tested the changes, everything works as expected.

Propagate from epic:
/label docs-approved
/label px-approved

**Pull request merged**